transcriptic/commands.py

```diff
@@ -1476,6 +1476,72 @@ def run_protocol(api, manifest, protocol, inputs, view=False, dye_test=False):
             return
 
 
+def execute(
+    autoprotocol,
+    api,
+    workcellid,
+    deviceset,
+    timelimit,
+    partitiongroupsize,
+    partitionhorizon,
+    partitioningswapdeviceid,
+):
+    # Get the autoprotocol
+    autoprotocol_str = autoprotocol.read()
+    try:
+        autoprotocol_dict = json.loads(autoprotocol_str)
+    except json.decoder.JSONDecodeError as err:
+        print("Error decoding autoprotocol json: ")
+        print(err)
```

Comment:
generally we should use `click.echo` for printing. For errors, we should use `click.error`. We could also explicitly return an error code